viewing_party/party.py

```diff
@@ -0,0 +1,95 @@
+def create_movie(title, genre, rating):
+    if title and genre and rating:
+        return {"title":title, "genre":genre, "rating":rating}
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data['watched'].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data['watchlist'].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data['watchlist']:
+        if movie['title'] == title:
+            user_data['watchlist'].remove(movie)
+            user_data['watched'].append(movie)
+    return user_data
+
+def get_watched_avg_rating(user_data):
+    if len(user_data['watched']) == 0:
+        return 0.0
+    else:
+        total = 0
+        for movie in user_data['watched']:
+            total += movie['rating']
+        return total/len(user_data['watched'])
+
+def get_most_watched_genre(user_data):
+    genre_count = {}
+    most_watched = None
+    if len(user_data['watched']) == 0:
+        return None
+    for movie in user_data['watched']:
+        if movie['genre'] in genre_count:
+            genre_count[movie['genre']] += 1
+        else:
+            genre_count[movie['genre']] = 1
+    for key, value in genre_count.items():
+        if not most_watched or genre_count[most_watched] < value:
+            most_watched = key
+    return most_watched
```

Comment:
this works great! Another way could be `most_watched = max(genre_count, key = genre_count.get) return(most_watched)`